### DISCUSSION TITLE
Nextflow history lock; disk quota exceeded

### DISCUSSION THREAD

**Discussion by zillurbmb51** [nextflow-io/nextflow#3327] [Q&A]
Hi, I am getting an error. `History lock; disk quota exceeded`

Any help?

Attached are my logs.
[nextflow_log2.txt](https://github.com/nextflow-io/nextflow/files/9898924/nextflow_log2.txt)
[nextflow_log1.txt](https://github.com/nextflow-io/nextflow/files/9898925/nextflow_log1.txt)
![nextflow_disk_quota_exceeded_Screenshot from 2022-10-31 08-19-33](https://user-images.githubusercontent.com/12833907/198954330-2ef85b5d-f7f9-4e1f-ad0c-8784cc3301dd.png)

**Comment by pditommaso** [comment]
`disk quota exceeded` means there's no more space in your storage. Please contact your sysadmin for details
